docs/CUSTOM_BOARDS_TARGETS.md

```diff
@@ -30,6 +30,17 @@ The suggested procedure is simple and makes it easier to update the code base by
     - Note: The device name can be 20 characters max.
 5. Modify the target's device_conf entry and hal file to match the features of your custom board - further detailed below.
 
+The advanced way:
+
+Detailed [here](https://github.com/olliw42/mLRS/blob/main/mLRS/CREATE_TARGET_INSTRUCTIONS.md)
+
+## Enabling Debug for a target
+
+Follow the 'advanced way' for creating a new target as linked above, with the following exception:
```

Comment:
based on the comment elsewhere, maybe

For a target which was created by following the 'advanced way', which resulted in a 'Relase' target, a 'Debug' target can be added by these steps:
- follow step 10, but click 'Debug' instead of 'Release'
- follow steps 12, 13 and 16 to the word
- if the target uses USB, then in addition follow step A1.2 to the word

Reply:
Reworked slightly, believe I captured the necessary steps.